### ISSUE TITLE
Option to nuke all TM:PE data in the savegame

### ISSUE THREAD

**Issue by aubergine10** [krzychu124/Cities-Skylines-Traffic-Manager-President-Edition#281]
Don't know whether this already exists or not, but it would be useful to have an option that nukes all TM:PE data from a savegame.

**Comment by DaEgi01** [comment]
yes please! :)

**Comment by aubergine10** [comment]
Another use case for this feature - if user changes driving direction mid-game: https://steamcommunity.com/sharedfiles/filedetails/?id=1742087656

**Comment by aubergine10** [comment]
While doing code review on PR #382 , noticed there's a DEBUG build option in mod settings to clear road speeds. So possibly there might be other similar features that we could expose at some point?

`resetSpeedLimitsBtn` in `TLM/TLM/State/Options.cs`

**Comment by aubergine10** [comment]
Maybe it a "Delete customisations" button could show a popup dialog with checkboxes (all selected by default) for what to delete:

> Which road customisations would you like to permanently delete?
>  
> (o) Speed limits
> (o) Lane arrows
> (o) Lane connectors
> (o) Vehicle restrictions
> (o) Parking restrictions
> (o) Junction restrictions
> (o) Priority signs
>  
> \[Cancel\] \[Delete Selected\]

**Comment by brunoais** [comment]
About the text, Instead of a "delete", I'd use a "disable". That gives a better impression that they will easily come back

**Comment by krzychu124** [comment]
@brunoais the key here is to delete those settings, not just disable. Currently you can disable any feature you want(maintenance tab in mod options) but settings stored inside savegame won't be removed then.

**Comment by aubergine10** [comment]
Yup, krzychu is correct: The goal here is to completely delete the settings from save game. This is useful if settings are corrupt, or the user just wants to reset everything to vanilla, etc.

**Comment by brunoais** [comment]
Ups. I mixed up PRs. Nvm.

**Comment by aubergine10** [comment]
Slightly different idea for resetting settings...

In **Maintenance** tab in mod settings, we have ability to disable most tools, we could maybe put reset buttons next to them?

![image](https://user-images.githubusercontent.com/1386719/74896249-b9bdba80-538b-11ea-985c-65d6bd6c204d.png)

Issues with approach: Some tools not listed (eg. lane arrows), and also a confirm dialog would be needed for each button which would be PITA if you wanted to reset everything... So probably a bad idea overall compared to previous ideas mentioned in other comments above..
